src/entities/cohort-member.entity.ts

```
@@ -19,4 +19,10 @@ export class CohortMember {
 
   @Column({ type: 'uuid', nullable: false })
   AcademicYearID: string;
+
+  @Column({ type: 'timestamp', nullable: false, default: () => 'CURRENT_TIMESTAMP' })
```

ESLint

[error] Replace `·type:·'timestamp',·nullable:·false,·default:·()·=>·'CURRENT_TIMESTAMP'` with `⏎····type:·'timestamp',⏎····nullable:·false,⏎····default:·()·=>·'CURRENT_TIMESTAMP',⏎·`

(prettier/prettier)